### ISSUE TITLE
status.client vs gateway's own web interface

### ISSUE THREAD

**Issue by laoshaw** [openNDS/openNDS#144]
status.client takes gateway's 80 port, some gateway's own web ui uses 80 port too,  how do they work together, am I missing something?

**Comment by bluewavenet** [comment]
@laoshaw 

> gateway's own web ui uses 80 port too

By default Luci uses port 80, but openNDS captures this port.
To access Luci run the following commands:
```
uci set uhttpd.main.listen_http='0.0.0.0:2080'
uci commit uhttpd
service uhttpd restart
```
Luci can then be accessed on port 2080
You can of course choose any unused port instead of 2080

Note: for security reasons it is probably sensible to at least change the Luci port even if you are not using openNDS. This reduces the chance of anyone accidentally finding Luci. If you intend to set up a public captive portal, it would be prudent to keep Luci disabled until needed for any admin work even if it is on some random port because it is not difficult to scan for it with simple tools.

Thanks for bringing this up, It would be a good addition to the documentation :-D

**Comment by laoshaw** [comment]
Thanks

**Comment by rc452860** [comment]
> @laoshaw
> 
> > gateway's own web ui uses 80 port too
> 
> By default Luci uses port 80, but openNDS captures this port.
> To access Luci run the following commands:
> 
> ```
> uci set uhttpd.main.listen_http='0.0.0.0:2080'
> uci commit uhttpd
> service uhttpd restart
> ```
> 
> Luci can then be accessed on port 2080
> You can of course choose any unused port instead of 2080
> 
> Note: for security reasons it is probably sensible to at least change the Luci port even if you are not using openNDS. This reduces the chance of anyone accidentally finding Luci. If you intend to set up a public captive portal, it would be prudent to keep Luci disabled until needed for any admin work even if it is on some random port because it is not difficult to scan for it with simple tools.
> 
> Thanks for bringing this up, It would be a good addition to the documentation :-D

can reuse 80 port? I see the options below：
```
option gatewayfqdn 'my.status'
Warning - if set, services on port 80 of the gateway will no longer be accessible (eg Luci AdminUI)
```
and I set the `option gatewayfqdn 'disable'`, it's can't not access luci admin ui  still.

**Comment by bluewavenet** [comment]
@rc452860 
This is a known bug and is fixed in the upcoming v9.3.1 release. Thanks anyway for mentioning it.
